app/views/results/marker/_marker_summary.html.erb

```diff
@@ -16,7 +16,7 @@
 
   <div class='summary_block'>
     <span><strong><%= t('results.subtotal') %></strong>:</span>
-    <span id='current_subtotal_div'> <%= result.get_subtotal %></span>
+    <span id='current_subtotal_div'> <%= result.get_subtotal(user_visibility: @result.is_a_review? ? :peer : :ta) %></span>
```

Comment:
It's a bit better to move the `is_a_review` check directly into the `Result#get_subtotal` method. This might mean we can remove the `user_visibility` parameter from this method altogether (check other usages).